Dockerfile

```diff
@@ -41,10 +37,15 @@ RUN pip freeze 2>/dev/null > requirements.installed \
         || ( echo "!! ERROR !! requirements.txt defined different packages or versions for installation" \
                 && exit 1 ) 1>&2
 
-COPY static /code/static
-COPY templates /code/templates
-COPY truewiki /code/truewiki
-RUN mkdir /data
+# NOTE this volume stores your pages be sure to link it and make it persistent. Don't forget to create the subfolders manually : /data/File, /data/Category, /data/Page, /data/Template
+VOLUME /data 
+# NOTE this volume may not be linked except if you want to use a tmpfs volume as a cache
+VOLUME /cache
+# NOTE link this volume to your source code folder
+VOLUME /code
```

Comment:
`/code` should not be in a volume. If you want to volume mount over it, that is fine, but this Dockerfile is not meant to do constant development. It is meant to build and run. Otherwise it is vastly better to create a virtualenv locally and run it outside the docker.